build.gradle

```diff
@@ -32,18 +32,22 @@ repositories {
 		name "RX14 Repository"
 		url 'http://mvn.rx14.co.uk/local/'
 	}
+    mavenLocal()
 }
 
 dependencies {
 	compile 'org.ow2.asm:asm:7.1'
 	compile 'org.ow2.asm:asm-commons:7.1'
 	compile 'org.ow2.asm:asm-tree:7.1'
 	compile 'org.ow2.asm:asm-util:7.1'
-	compile 'net.fabricmc:tiny-mappings-parser:0.1.1.8'
+	compile 'net.fabricmc:tiny-mappings-parser:0.1.2'
 	implementation group: 'org.checkerframework', name: 'checker-qual', version: '2.5.7'
-	implementation('cuchaz:enigma:0.14.0.120') {
-		exclude group: 'org.ow2.asm'
-	}
+    implementation('cuchaz:enigma:0.14.299') {
```

Comment:
Is this version right?

Reply:
ignore versions for now, everything is built locally.